vaporetto_rules/Cargo.toml

```diff
@@ -10,7 +10,11 @@ repository = "https://github.com/daac-tools/vaporetto"
 readme = "README.md"
 keywords = ["japanese", "analyzer", "tokenizer", "morphological"]
 categories = ["text-processing", "no-std"]
+resolver = "2"
 
 [dependencies]
 unicode-segmentation = "1.9.0"  # MIT or Apache-2.0
-vaporetto = { path = "../vaporetto", version = "0.4.0", default-features = false }  # MIT or Apache-2.0
+vaporetto = { path = "../vaporetto", version = "0.5.0", default-features = false, features = ["alloc"] }  # MIT or Apache-2.0
+
+[dev-dependencies]
+vaporetto = { path = "../vaporetto", version = "0.5.0" }  # MIT or Apache-2.0
```

Comment:
Needed?

Reply:
Yes.
vaporetto_rules only uses `alloc` feature of vaporetto, but the document uses `std` feature to show examples.

Reply:
I see.